src/components/MyFeed/index.js

```diff
@@ -23,12 +26,15 @@ const MyFeed = () => {
             Explore top rated Organizations and find the <b>Codelabz</b> you are
             looking for
           </p>
+          <div item xs={12}>
+            <Carousel />
+          </div>
         </Grid>
 
-        <Grid item xs={12}>
-          {" "}
-          <ExploreOrgs />
-        </Grid>
+        {/* <Grid item xs={12} > */}
```

Comment:
Is there any reason why these lines are commented out?

Reply:
actually those are previous view thats why commented them out , Now everyone approved this design I will remove those comment

Reply:
fixed